org.lflang.cli/src/org/lflang/cli/Lff.java

```diff
@@ -0,0 +1,351 @@
+/**
+ * Stand-alone version of the Lingua Franca formatter (lff).
+ */
+
+package org.lflang.cli;
+
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.xtext.util.CancelIndicator;
+import org.eclipse.xtext.validation.CheckMode;
+import org.eclipse.xtext.validation.IResourceValidator;
+import org.eclipse.xtext.validation.Issue;
+
+import org.lflang.LFRuntimeModule;
+import org.lflang.LFStandaloneSetup;
+import org.lflang.ast.ToLf;
+import org.lflang.util.FileUtil;
+
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+import com.google.inject.Provider;
+
+/**
+ * Standalone version of the Lingua Franca formatter (lff).
+ * Based on lfc.
+ *
+ * @author {Marten Lohstroh <marten@berkeley.edu>}
+ * @author {Christian Menard <christian.menard@tu-dresden.de>}
+ * @author {Billy Bao <billybao@berkeley.edu>}
+ */
+public class Lff {
+
+    /// current lff version as printed by --version
+    private static final String VERSION = "0.2.2-SNAPSHOT";
```

Comment:
Rather than replicating this string, we should move this elsewhere (probably a class in the root of the `org.lflang` package) and reference it from both `Lff` and `Lfc`.

Reply:
Not sure where's a good place to put this. Would org.lflang.FileConfig work?

Reply:
Not sure that's the right place either. I imagine it would be useful to gather strings in a single file to make it easy to style them a certain way and eventually do internationalization. The version number could be in a class like that. This might be the right time to start thinking about this. I believe Java uses property files and `ResourceBundle` classes to deal with collections of strings...

Reply:
Did that!